output/telegram.go

```diff
@@ -126,7 +136,7 @@ func (t *TelegramOutput) sendAlertmanagerImage(spanCtx sreCommon.TracerSpanConte
 
 	query, ok := alert.Labels[t.options.AlertExpression]
 	if !ok {
-		err := errors.New("No alert expression")
+		err := errors.New("No alert expression.")
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings
"Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context."

Reply:
there will be no following context in exactly this case.